src/include/daos/mem.h

```diff
@@ -153,10 +153,23 @@ typedef enum {
 	UMEM_CLASS_PMEM,
 	/** persistent memory but ignore PMDK snapshot */
 	UMEM_CLASS_PMEM_NO_SNAP,
+	/** ad-hoc memory */
+	UMEM_CLASS_AD,
 	/** unknown */
 	UMEM_CLASS_UNKNOWN,
 } umem_class_id_t;
 
+#define UTX_PRIV_SIZE	(512)
```

Comment:
I think 256 bytes is enough for ad_tx

Reply:
OK, more info possibly will be added to ad_tx later, we can reduce the size later.